packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -62,15 +62,24 @@ export function isLinux() {
   return os.type() === "Linux";
 }
 
+export interface TeamsAppTelemetryInfo {
+  appId: string;
+  tenantId: string;
+}
+
 // Only used for telemetry when multi-env is enabled
-export function getTeamsAppIdByEnv(env: string) {
+export function getTeamsAppTelemetryInfoByEnv(env: string): TeamsAppTelemetryInfo | undefined {
   try {
     const ws = ext.workspaceUri.fsPath;
 
     if (isValidProject(ws)) {
       const result = environmentManager.getEnvStateFilesPath(env, ws);
       const envJson = JSON.parse(fs.readFileSync(result.envState, "utf8"));
-      return envJson[PluginNames.APPST].teamsAppId;
+      const aadState = envJson[PluginNames.APPST];
```

Comment:
Why call this `aadState` since it is more related to App Studio/Teams App?

Reply:
Sorry that's a typo. 😂

Reply:
fixed